README.md

```diff
@@ -17,7 +17,14 @@ The following are required in order to build this project:
 * cmake >= 3.2.0
 * Qt 5 (`qt5-default` on Ubuntu)
```

Comment:
Since you're providing the ubuntu package names below, we can probably remove the "(`pkgname` on ubuntu)" next to the dependencies here.